src/app/app.css

```
@@ -0,0 +1,31 @@
+.enter-animation {
+  display: block;
+  animation: slideIn 200ms ease-in forwards;
+}
+
+@keyframes slideIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)